README.md

```diff
@@ -135,6 +135,25 @@ To use salt-lint with [pre-commit](https://pre-commit.com),  just add the follow
 
 # Rules
 
+
+## List of rules
+
+| Rule                                                 | Description                                                                  |
+|:-----------------------------------------------------|:-----------------------------------------------------------------------------|
+| [201](https://github.com/warpnet/salt-lint/wiki/201) | Trailing whitespace                                                          |
+| [202](https://github.com/warpnet/salt-lint/wiki/202) | Jinja statement should have spaces before and after: `{% statement %}`       |
+| [203](https://github.com/warpnet/salt-lint/wiki/203) | Most files should not contain tabs                                           |
+| [204](https://github.com/warpnet/salt-lint/wiki/204) | Lines should be no longer that 160 chars                                     |
+| [205](https://github.com/warpnet/salt-lint/wiki/205) | Use ".sls" as a Salt State file extension                                    |
+| [206](https://github.com/warpnet/salt-lint/wiki/206) | Jinja variables should have spaces before and after `{{ var_name }}`         |
+| [207](https://github.com/warpnet/salt-lint/wiki/207) | File modes should always be encapsulated in quotation marks                  |
+| [208](https://github.com/warpnet/salt-lint/wiki/208) | File modes should always contain a leading zero                              |
+| [209](https://github.com/warpnet/salt-lint/wiki/209) | Jinja comment should have spaces before and after: `{# comment #}`           |
+| [210](https://github.com/warpnet/salt-lint/wiki/210) | Numbers that start with `0` should always be encapsulated in quotation marks | 
+| [211](https://github.com/warpnet/salt-lint/wiki/211) | `pillar.get` or `grains.get` should be formatted differently                 |
+| [212](https://github.com/warpnet/salt-lint/wiki/212) | Most files should not contain irregular spaces                               |
+
```

Comment:
OK, this part is a non-issue, since the links lead to the wiki.  It's funny, I didn't see the link immediately when looking at this from `View file`.  Perhaps I wouldn't have missed it as easily if the main statement was the link?  More noticeable, perhaps.

Reply:
So perhaps the rule numbers in bold and the main text as the links?  Just some thoughts for you to consider, feel free to ignore this suggestion.

Reply:
Thanks @myii for your suggestion, I will leave this one open for @jbouter to decide!

Reply:
I've based the table on how [hadolint](https://github.com/hadolint/hadolint/#rules) does it, and I find it quite easy and nice to use. If you could, please have a look how hadolint's table feels to you, @myii, and tell me. :-) If acceptable, I'd like to implement it how I've currently designed it.

Reply:
@jbouter Sure, keep it that way.  I just didn't see the link at first, that's all!  But I'm sure it will be fine.

The one thing that probably needs adjusting is whether to use leading and trailing bars (`|`) or not.  So the line that has been modified should follow the same system as the rest of the table.

Thanks for providing this feature, by the way.

Reply:
Hey @myii, thanks for your review. I've modified the table to be simpler to edit as per your suggestion. Could you review it once more?

Reply:
@jbouter Sure, it looks nice and tidy, thanks!